docker/01-init-user.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+set -e
+
+psql -v ON_ERROR_STOP=1 --username postgres --dbname postgres <<-EOSQL
```

Comment:
What about [geostore database initialization](https://mapstore.readthedocs.io/en/v2020.02.01/developer-guide/database-setup/)? Is it done automatically?

Reply:
the Geostore database initialization was resolve here https://github.com/geosolutions-it/MapStore2/pull/6490/commits/93b4066b4ac0b464374b3e97225ec8921c510aaf

Reply:
no, the database is empty

Reply:
Database initialization script here:
https://github.com/geosolutions-it/geostore/blob/master/doc/sql/002_create_schema_postgres.sql

Reply:
resolved here https://github.com/geosolutions-it/MapStore2/pull/6490/commits/a6c3879dbce8330ffe5431dddd1dc5d59735452f

Reply:
please download the scripts from the geostore repo instead of copying the content